graphql/event.go

```diff
@@ -182,6 +182,7 @@ type TaskCompletedEvent struct {
 	Phase     int              `json:"phase"`
 	Task      string           `json:"task"`
 	Points    *uint64          `json:"points,omitempty"`
+	Override  bool             `json:"override"`
```

Comment:
Don't you need to add the `omitempty` attribute since old event already registered doesn't include this property ? Or maybe in case of a boolean, missing property is set to false by default ?  

This is in case of we need to replay all event.

Reply:
Good point, I had a strong hesitation about that!

Actually the unmarshal of old events works without the `omitempty` tag by setting the default boolean value `false` as you intuited, so we have no misbehavior.

As I prefer having the `override` flag present in the marshalled event i decided to not set the `omitempty` tag, if you're good with it?

Reply:
Ok ! If the unmarshaling of old event work, no problem for me to let it like this ! 👍